src/_bundle.es6.js

```diff
@@ -0,0 +1 @@
+export { default } from './index.js';
```

Comment:
All generated es6 output (`dist/js/helix-ui.module.js` for consumption directly in modern browsers) starts with this file.

We might have been able to just as easily use `src/index.js` as the entry point, but this makes it explicit that we have two `_bundle*` entry points.